### PR TITLE
Code Clean-Ups [See issues in https://github.com/DataSlingers/clustRviz/issues/116]

### PR DESCRIPTION
**Pull request by dansenglund**

### CODE REVIEW

**Comment by michaelweylandt** [comment]
Good news - CI seems to be more-or-less back to normal. For some reason the Windows CI is running about 2 hours slower than everything else though.... Don't know if that's an old compiler problem (Eigen is pretty taxing to use) or something else.

**Comment by michaelweylandt** [comment]
@dansenglund Can we add some (very basic) tests for `MatrixColProx` as well. (Same file as the tests for `MatrixRowProx` should be fine) 

They don't need to be as extensive as what we have for the row-wise prox. A few variants on the identity `t(MatrixColProx(t(X))) == MatrixRowProx(X)` should be fine.

File: src/clustering_impl.h [hunk 1]
```diff
@@ -121,13 +117,9 @@ class ConvexClustering {
   }
 
   bool admm_converged(){
-    return ((V - V_old).squaredNorm() + (Z - Z_old).squaredNorm() < 1e-7);
+    return (scaled_squared_norm(V - V_old) + scaled_squared_norm(Z - Z_old) < EXACT_STOP_PRECISION);
```
Comment (michaelweylandt) [inline src/clustering_impl.h:120]: Dumb question: Is there are reason we aren't checking `U - U_old` here? I can't think of one...

Comment (dansenglund) [reply src/clustering_impl.h:120]: No reason not to I suppose. My Rice email is valid for another year or so, which I imagine will be as long as I'm relevant to the project

Comment (dansenglund) [reply src/clustering_impl.h:120]: Wait. There is no U_old in clustering, which explains why it's omitted here

Comment (michaelweylandt) [reply src/clustering_impl.h:120]: Oh - interesting... I guess that was a memory saving thing at some point. I suppose it's harmless - at some point maybe I'll play around and see if it's worth dropping `U_old` from biclustering as well then.

File: src/clustRviz_base.h [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 #define CLUSTRVIZ_STATUS_UPDATE_TIME_SECS 0.1 // Every 0.1s
 #define CLUSTRVIZ_STATUS_WIDTH_CHECK 20 // Every 20 status updates * 0.1s => every 2s
+#define EXACT_STOP_PRECISION 1e-10 //Stop when cellwise diff between iters < val
```
Comment (michaelweylandt) [inline src/clustRviz_base.h:10]: Can we "namespace" this like the other `#define`s? `CLUSTRVIZ_EXACT_STOP_PRECISION`

**Pull request merged**